packages/react-tabs/Spec.md

```diff
@@ -316,51 +344,61 @@ export type TabListProps = ComponentProps<TabListSlots> & {
 
 ### From v8
 
-- Replace occurances of `<Pivot>` with `<TabList>`
-- Replace occurances of `<PivotItem>` with `<Tab>`
-- Replace `<PivotItem>` content with handling onTabSelected to show/hide associated content.
-- Move PivotItem.headerText to be the content of Tab
+1. Replace occurances of `<Pivot>` with `<TabList>`
+2. Replace occurances of `<PivotItem>` with `<Tab>`
+3. Replace `<PivotItem>` content with handling onTabSelected to show/hide associated content.
+4. Move PivotItem.headerText to be the content of Tab
+5. Subscribe to onTabSelected to show/hide content when a tab is selected.
 
 ### From v0
 
-- Replace use of `<Menu>` with `<TabList>` (full rewrite)
+1. Replace use of `<Menu>` with `<TabList>`
+2. Replace items data with Tab instances, writing a `map` function as needed.
 
 ## Behaviors
 
 ### Selection
 
-The TabList will provide single-selection of a tab.
+TabList provides single select of a tab.
 
-### Programmatic Selection
+Programmatic
 
-The TabList.selectedKey can control the currently selected tab.
-If the tab to be selected is in the overflow, it will be moved into the set of visible tabs.
+- The TabList.selectedKey can control the currently selected tab.
 
-> TODO: Do we need an announce here?
+Mouse/Touch
 
-### Overflow
+- Clicking the tab selects it.
 
-The overflow part will screen read as a button with text indicating to click for a list of other available tabs.
+Keyboard
 
-### Mouse/Touch
+- Tablist focus traps the and allows focus of individual tabs.
+- Arrow keys move focus forward/backward.
+- The arrow keys are based on layout. Horizontal: left/right, Vertical: up/down.
```

Comment:
let's have up+left move focus to previous and down+right move to next, independently of the orientation. That way users who cannot see the visual alignment do not have to care about the orientation.